src/components/options.tsx

```diff
@@ -66,6 +66,10 @@ const MarkdownPanel: React.FC = () => {
 	);
 };
 
+const CssPanel: React.FC = () => {
+	return <div />;
+};
+
```

Comment:
You do not have to return an empty div rather you could return null if there were no options